src/main/java/com/cpdaimler/bootstrap/CPBootstrap.java

```diff
@@ -122,4 +137,21 @@ private void initCars() {
         carRepository.save(c1);
 
     }
+
+
+    private void getRandomLatLon(Shift shift) {
+
+        Double random = Math.random() * 0.1;
```

Comment:
Use ThreadLocalRandom instead @jonathanba

Reply:
Ah... just read about it, thank you. First time I heard about. It seems like there is just a small performance advantage (https://www.baeldung.com/java-thread-local-random ). I'll patch it next time.